tooling/comparable_patch.sh

```diff
@@ -0,0 +1,275 @@
+#!/bin/bash
```

Comment:
Broad comment to split this into named functions

Reply:
yep done